game-core/src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -84,10 +84,7 @@ public void load(final File file) {
         setGameData(newData);
       }
     } catch (final Exception e) {
-      log.log(Level.SEVERE,
-          String.format("Error loading game file: %s, %s",
-              file.getAbsolutePath(), e.getMessage()),
-          e);
+      log.log(Level.SEVERE, "Error loading game file: " + file.getAbsolutePath(), e);
```

Comment:
Removed a redundant copy of the error message.  The log formatter will include it in the final log message.